src/main/File/Services/UploadFileService.java

```diff
@@ -68,8 +68,7 @@ public Message executeAndGetResponse() {
     } else if (fileStream == null) {
       return FileMessage.INVALID_FILE;
     } else if (fileType.isPDF()) {
-      if (!fileContentType.equals("application/pdf")
-          && !fileContentType.equals("application/octet-stream")) {
+      if (!fileContentType.equals("application/pdf")) {
```

Comment:
The upload file service is used with profile pictures, so we need to support jpgs as well.

Reply:
Ok thanks, yea I knew that I should have also added images here as a file type, to make it consistent with the other uploaders - sorry for not doing that